lib/pina/models/sales_order.rb

```diff
@@ -0,0 +1,98 @@
+module Pina
+  module Models
+    class SalesOrder
+      include Virtus.model
+
+      attribute :created_at
+      attribute :updated_at
+      attribute :creator
+      attribute :creator_id
+      attribute :modifier
+      attribute :modifier_id
+      attribute :accounting_address
+      attribute :accounting_address_id
+      attribute :buyer_address
+      attribute :buyer_address_id
+      attribute :document_recipient_address
+      attribute :document_recipient_address_id
+      attribute :delivery_address
+      attribute :delivery_address_id
+      attribute :contact
+      attribute :currency
+      attribute :currency_id
+      attribute :date_of_receipt
+      attribute :delivery_method
+      attribute :department
+      attribute :department_id
+      attribute :percent_discount
+      attribute :external_id
+      attribute :fullfilled_on
+      attribute :group_id
+      attribute :invoicing_method
+      attribute :status
+      attribute :order
```

Comment:
Hele ja ted moc nevim, tyhle atributy znamenaj to co se ti vraci v odpovedi nebo to co posilas? Jinak api nam vraci nasledujici: Nicmene napr. u contract to neni cisty atribut jakoze string treba ale je to objekt, ve kterem je "contract_id"

    "accounting_address"
    "buyer_address"
    "contact"
    "currency"
    "date_of_receipt"
    "delivery_address"
    "delivery_method"
    "department"
    "percent_discount"
    "document_recipient_address"
    "external_id"
    "fullfilled_on"
    "group_id"
    "invoicing_method"
    "status"
    "contract"
    "shipper_id"
    "shipping_method"
    "specific_symbol"
    "payment_method"
    "order_id"
    "complete_upon_execution"
    "items",
    "created_at"
    "creator"
    "updated_at"
    "modifier"